src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2563,6 +2563,12 @@ public override BoundNode VisitImplicitReceiver(BoundImplicitReceiver node)
             return null;
         }
 
+        public override BoundNode VisitEqualsValue(BoundEqualsValue node)
```

Comment:
I copied the form of some other overriding Visit*** methods. Hopefully this is close to correct.